app/templates/search/_briefs_results.html

```diff
@@ -18,7 +18,7 @@ <h2 class="search-result-title">
             {{ brief.lotName }}
         </li>
         <li class="search-result-metadata-item">
```

Comment:
We should only include the `li` if it has content to show. Some of the briefs won't have a specialist role as they are not on the correct lot so probably need to wrap this `li` in some kind of if statement